lib/rubocop/cop/ezcater/rspec_match_ordered_array_not_eq.rb

```diff
@@ -0,0 +1,44 @@
+module RuboCop
+  module Cop
+    module Ezcater
+      # Enforce use of `match_ordered_array` matcher instead of using `eq` matcher
+      #
+      # @example
+      #
+      #   # good
+      #   expect(foo).to match_ordered_array([1, 2, 3])
+      #   expect(foo).to match_ordered_array [1, 2, 3]
+      #
+      #   # bad
+      #   expect(foo).to eq([1, 2, 3])
+      #   expect(foo).to eq [1, 2, 3]
+      class RspecMatchOrderedArrayNotEq < Cop
+        MATCH_ORDERED_ARRAY = "match_ordered_array".freeze
+        MSG = "Use the `match_ordered_array` matcher instead of `eq` when comparing collections".freeze
```

Comment:
Is it worth mentioning the `ezcater_matchers` gem when recommending this?

Reply:
^^ I think we should. This could be a confusing google, otherwise.

Reply:
totally!